<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>charge</class>
 <widget class="QDialog" name="charge">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="char_money">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>충전할 금액 입력(10,000원 이상)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="charge_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>40</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>충전</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>97</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>캐시 충전시 마일리지 지급!</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>166</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>결제금액</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>135</x>
     <y>217</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>할인된 금액</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>263</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>297</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>총 액</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pay">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>347</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>결제</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="all_text">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>179</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="fin_text">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>290</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="sale_text">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>220</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
